docs/source/contributor-guide/index.md

```diff
@@ -95,7 +95,7 @@ Compiling DataFusion from sources requires an installed version of the protobuf
 On most platforms this can be installed from your system's package manager
 
 ```
-$ apt install -y protobuf-compiler
+$ sudo apt install -y protobuf-compiler
 $ dnf install -y protobuf-devel
```

Comment:
I wonder if we should add `sudo` to the other similar commands (like `dnf` and `pacman`) 🤔

Reply:
I have tested it on my Linux machine. Pacman and dnf can be used without sudo.
But apt install -y protobuf-compiler seems to require sudo under ordinary users.

Reply:
But the dnf command also seems to require apt installation. Do you need to add something?

Reply:
Let's start with this change and we can always update the instructions later if we find something that could make it clearer

Thanks again @caicancai